dynawo/3rdParty/CMakeLists.txt

```diff
@@ -521,11 +542,18 @@ else()
 
       CONFIGURE_COMMAND   <SOURCE_DIR>/bootstrap.sh
                           --without-icu
-                          --with-libraries=filesystem,program_options,serialization,system,log,iostreams
+                          --with-libraries=date_time,filesystem,program_options,serialization,system,log,iostreams,log_setup
```

Comment:
Same remark about c++11 only

Reply:
It was a question, should I change something in the file ? It makes it simpler, that is why I choose that way.

Reply:
Yes, but we install dependencies that are useless in cpp98, so IMHO we should separate both cases...
@gautierbureau what is your opinion on this?

Reply:
I would say I could live with installing some useless libraries for the moment to avoid adding a new big block of if/else cx11_enabled that would add some confusion in already large file. And besides us nobody should install in cpp98.

Reply:
You are missing test in the libraries. And log_setup is not a valid library name: I get this error
error: wrong library name 'log_setup' in the --with-<library> option.

Reply:
done anyway